### PR TITLE
Restructured readme, moved tutorials to sample readme

### PR DESCRIPTION
**Pull request by SaschaWillems**
## Description

This PR moves the tutorials from the main repo readme to the samples readme. Instead of a single listing of all available tutorials, available tutorials are now associated with their accompanying sample.

## General Checklist:

Please ensure the following points are checked:

- [n/a] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Code-Style)
- [x] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#Copyright-Notice-and-License-Template)
- [n/a] I have commented any added functions (in line with Doxygen)
- [n/a] I have commented any code that could be hard to understand
- [n/a] My changes do not add any new compiler warnings
- [n/a] My changes do not add any new validation layer errors or warnings
- [n/a] I have used existing framework/helper functions where possible
- [n/a] My changes build on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/master/CONTRIBUTING.md#General-Requirements)
- [n/a] My changes do not add any regressions
- [n/a] I have tested every sample to ensure everything runs correctly
- [x] This PR describes the scope and expected impact of the changes I am making

### CODE REVIEW

**Review by r-potter — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -57,42 +57,13 @@ Additionally you may find the following links useful:
 ## Samples
 - [Listing of all samples available in this repository ](./samples/README.md)
 
-## Tutorials
```
Comment (JoseEmilio-ARM) [inline README.md:60]: Need to also update the table of contents at the top

**Review by aurolou — DISMISSED** [review]

**Comment by JoseEmilio-ARM** [comment]
shall we also remove the README files in samples/api and samples/extensions since those will be going out of date?

File: README.md (hunk 1 above)
Comment (SaschaWillems) [reply README.md:60]: I have removed the tutorial links from the TOC and paragraphs.

Comment (JoseEmilio-ARM) [reply README.md:60]: cool thanks! but shouldn't we replace it with a link to the new General information section?

Comment (SaschaWillems) [reply README.md:60]: Somehow missed that. Just pushed a commit that adds the general information section to the TOC.

**Review by JoseEmilio-ARM — APPROVED** [review]

**Review by carlosadcq — APPROVED** [review]

**Review by gary-sweet — APPROVED** [review]

**Pull request merged**